test/integration/utils.test.js

```diff
@@ -226,4 +228,80 @@ describe(Support.getTestDialectTeaser('Utils'), function() {
       expect(Utils.singularize('status')).to.equal('status');
     });
   });
+
+  describe('Sequelize.fn', function() {
```

Comment:
While an integration test might not be strictly needed, I think this is good to keep as an example of how the functionality can be used